rules/recording.go

```diff
@@ -38,6 +38,8 @@ type RecordingRule struct {
 	mtx sync.Mutex
 	// The health of the recording rule.
 	health RuleHealth
+	// Unix timestamp of last evaluation of rule
```

Comment:
Style nit: Dito, period at the end.

`time.Time` is not a Unix timestamp, so I'd drop the `Unix`.